### ISSUE TITLE
[1.5.1/Linux] Adding a second NM-16ESW slot to c3725 is useless

### ISSUE THREAD

**Issue by jean-christophe-manciot** [GNS3/dynamips#73]
Arping a node connected on the second slot from a node connected on the first slot does not get through the c3725 used as a switch.
The tests have been performed with all ports occupied on the first slot.
Wireshark shows that the ARP are received on the first slot but not transmitted on the second one.
Of course, no such issue between ports on the first slot.

**Comment by noplay** [comment]
Hi,
You have issues with all c3725 images?

**Comment by jean-christophe-manciot** [comment]
Yes, same issue with 2 different releases. Other information includes:
- one must be cautious about the fact that none of the ports of the 2nd slot is configured (speed, duplex), whereas all ports of 1st slot are. Configuring the second slot does not solve the issue.
- it is possible to ping 2 devices connected on the second slot
